### DISCUSSION TITLE
What is this applications module(?) in the bar shown in the wiki?

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [YaLTeR/niri#621] [Q&A]
Saw this with the 5 alactritty icons on top, what bar/module/etc is that? :eyes: 

![image](https://github.com/user-attachments/assets/5d578e0f-e0b4-4fd4-9b52-af3d235b9fed)

**Comment by YaLTeR** [comment]
That's just `wlr/taskbar`, I have it show only the icons.

**Reply by YaLTeR** [reply]
(in Waybar)
